lib/cdo/git_utils.rb

```diff
@@ -93,7 +93,7 @@ def self.git_revision_branch(branch)
   def self.get_branch_commit_merges(commit)
     commit_json = JSON.parse(open("https://api.github.com/repos/code-dot-org/code-dot-org/commits/#{commit}").read)
     commit_json['commit']['message'].match(/from code-dot-org\/(.*)\n\n/)[1]
-  rescue => _
+  rescue => _e
```

Comment:
another spot where I'm not sure we need to assign a variable. Does this or another linter require that?

Reply:
I definitely thought there was a linter that did require a variable here, but apparently not!